src/test/java/org/humancellatlas/ingest/schemas/SchemaServiceTest.java

```diff
@@ -28,12 +28,12 @@
     private SchemaScraper schemaScraper;
 
     @Test
-    public void testGetLatestSchemas() {
+    public void testGetLatestSchemasVersions() {
```

Comment:
Thanks for adding tests!

Reply:
That is the right thing to do, specially if it is easy to do. ;-)